app/main_test.go

```diff
@@ -126,6 +127,24 @@ func TestFormat(t *testing.T) {
 	}
 }
 
+func TestExclusionPatterns(t *testing.T) {
+	tbl := []struct {
+		msg  		string
+		result  	bool
+		ExcludeList []string
+	}{
+		{"All I hear is blah blah blah", true, []string{"blah"}},
```

Comment:
the match is regex, but tests don't have anything using regex match

Reply:
Added regex's to the tests